api/rs/slint/Cargo.toml

```diff
@@ -38,17 +38,32 @@ std = ["i-slint-core/std"]
 ## backend will not be operational
 backend-qt = ["i-slint-backend-selector/i-slint-backend-qt", "std"]
 
-## The GL backend uses the `winit` crate for the windowing system integration,
-## and the `femtovg` crate for the rendering. With this feature, all windowing
-## systems are supported. For a smaller build, omit this feature and select
-## one of the other specific `backend-gl-XX` features.
-backend-gl-all = ["i-slint-backend-selector/backend-gl-all", "std"]
-## Simliar to `backend-gl-all` this enables the GL backend but only with support for the
-## X Window System on Unix.
-backend-gl-x11 = ["i-slint-backend-selector/backend-gl-x11", "std"]
-## Simliar to `backend-gl-all` this enables the GL backend but only with support for the
-## Wayland window system on Unix.
-backend-gl-wayland = ["i-slint-backend-selector/backend-gl-wayland", "std"]
+## This feature is an alias for the `eventloop-winit` feature and `renderer-femtovg`
```

Comment:
Same comment as for the interpreter/Cargo.toml:  they should all be documented, or none.